### PR TITLE
Synced and updated jshint config w/ template

### PR DESCRIPTION
**Pull request by fhamon**
Take a look at http://jshint.com/docs/options/
for new options and what has been removed

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -76,29 +77,29 @@ module.exports = function fxGruntConfig (grunt) {
 				maxdepth: 5,
 				maxstatements: 30,
 				maxlen: 100,
-				//maxcomplexity: 10,
```
[the remark this answers is not in the document]
Comment (fhamon) [reply Gruntfile.js]: https://ci.appveyor.com/project/nitriques/framework-js-modules/build/1.6.x.36/job/1lvrgs9l2keg30a6

File: Gruntfile.js [hunk 2]
```diff
@@ -57,14 +57,15 @@ module.exports = function fxGruntConfig (grunt) {
...
 				bitwise: false,
 				camelcase: false,
 				curly: true,
 				eqeqeq: false, // allow ==
 				forin: true,
-				//freeze: true,
+				freeze: true,
 				immed: false, //
-				latedef: false, // late definition
+				latedef: true, // late definition
```
[the remark this answers is not in the document]
Comment (fhamon) [reply Gruntfile.js]: You can see it in action here 🌮 : https://ci.appveyor.com/project/nitriques/framework-js-modules/build/1.6.x.36/job/1lvrgs9l2keg30a6

[the remark this answers is not in the document]
Comment (nitriques) [reply Gruntfile.js]: Boolean sucks so much. Perfect.

[the remark this answers is not in the document]
Comment (fhamon) [reply Gruntfile.js]: hehehehe

File: Gruntfile.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (nitriques) [reply Gruntfile.js]: I know. But having 2 tools check the same thing is good: if one tool ever shits on us, we should have a backup. Was it causing problems ?

**Comment by fhamon** [comment]
@nitriques @PascalPiche What do you think about increasing `maxlen` from 100 to 110? 

In a lot of cases, lines to long often bleed out for 1 to 5 characters and I find it mildly infuriating to modify the line to something less developper frendly and more difficult to read.

**Comment by PascalPiche** [comment]
Still prefer 100 as it a well adopted standard

**Comment by fhamon** [comment]
Totally agree with you @PascalPiche we should never go over 100 but it's for the many exceptions where we go around ~105 and "fixing" the maxlength issue create visual noise in the code and become less readable.

**Comment by nitriques** [comment]
Let's keep it to 100 for now, please provide a seperate PR.

File: Gruntfile.js [hunk 3]
```diff
@@ -76,29 +77,29 @@ module.exports = function fxGruntConfig (grunt) {
...
 				globals: {
 					jQuery: true,
 					console: true,
 					App: true,
 					Loader: true,
 					_: true,
-					google: true
+					google: true,
+					module: true,
+					THREE: true,
```
[the remark this answers is not in the document]
Comment (nitriques) [reply Gruntfile.js]: Why not?

File: Gruntfile.js [hunk 4]
```diff
@@ -76,29 +77,29 @@ module.exports = function fxGruntConfig (grunt) {
...
+					TWEEN: true
```
[the remark this answers is not in the document]
Comment (fhamon) [reply Gruntfile.js]: TweenMax but way much lighter. Really cool w/ THREE

File: Gruntfile.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (nitriques) [reply Gruntfile.js]: Ok so we never used it, good then.

File: Gruntfile.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (nitriques) [reply Gruntfile.js]: I feel like both THREE and TWEEN could be dealt with on a per file basis, usign /* jshint global */ directive. Please remove them from both repos. Thanks @fhamon

[the remark this answers is not in the document]
Comment (nitriques) [reply Gruntfile.js]: @fhamon please remove

File: Gruntfile.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (nitriques) [reply Gruntfile.js]: @fhamon please remove

**Pull request merged**